contributions-guide/vss-extension.json

```diff
@@ -665,6 +665,50 @@
                 "uri": "context.html",
                 "action": "PR Details Tab"
             }
+        },
+        {
+            "id": "sample-pr-status-menu",
+            "type": "ms.vss-web.menu",
+            "description": "Status menu for status with genre 'fabricam-ci-service' and name 'ci-build'",
+            "targets": [
+                "ms.vss-code-web.pull-request-status-menu"
+            ],
+            "properties": {
+                "statusGenre": "fabricam-ci-service",
+                "statusName": "ci-build"
+            }
+        },
+        {
+            "id": "statusMenuItem_1",
+            "type": "ms.vss-web.action",
+            "description": "Executes custom action for the status",
+            "targets": [
+                "ms-samples.samples-contributions-guide.sample-pr-status-menu"
```

Comment:
Can you use a relative target here? e.g. `"targets": [ ".sample-pr-status-menu" ]` This extension is sometimes used as a starting point for extension developers, so they may change the publisher. Using the relative target ID, it doesn't matter what the publisher or extension name is.

Reply:
It makes sense. Updated to use relative targets.